graphicsmagick-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "graphicsmagick-sys"
-version = "0.1.2"
+version = "0.4.0"
```

Comment:
We might not need a new major release on sys crate since I did not modify any API of it.

Reply:
But build.rs is changed.

Reply:
I known what you mean, but it has released. :cry:

Reply:
It should be ok unless one depends on both graphicsmagick-sys 0.2.1 and 0.4.0